cdap-data-fabric/src/test/java/co/cask/cdap/data/hbase/HBaseTestBase.java

```diff
@@ -55,7 +55,16 @@ public String getZkConnectionString() {
 
   // Test startup / teardown
 
-  public abstract void startHBase() throws Exception;
+  public final void startHBase() throws Exception {
+    // Tune down the connection thread pool size
+    getConfiguration().setInt("hbase.hconnection.threads.core", 5);
+    getConfiguration().setInt("hbase.hconnection.threads.max", 10);

```

Comment:
These settings will only be effective when you call HConnection.getTable() to obtain table instances, which I don't think we really do anywhere.

We should set hbase.htable.threads.max to limit the number of threads in the thread pool constructed when instantiating HTable directly -- new HTable(Configuration, TableName).  I think this is the usage we have almost everywhere.